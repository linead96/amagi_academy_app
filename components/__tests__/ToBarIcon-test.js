import 'react-native'
import React from 'react'
import ToBarIcon from '../TabBarIcon';
import renderer from 'react-test-renderer';

test('renders an icon based on the name of the icon passed as props into it, in this test md-checkmark is passed as props.', () => {
    const tree = renderer.create(<ToBarIcon name={'md-checkmark'}/>).toJSON;
    expect(tree).toMatchSnapshot();
})