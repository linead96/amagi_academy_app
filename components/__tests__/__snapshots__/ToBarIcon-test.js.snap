// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an icon based on the name of the icon passed as props into it, in this test md-checkmark is passed as props. 1`] = `[Function]`;
